### DISCUSSION TITLE
Styling like the wiki.js docs

### DISCUSSION THREAD

**Discussion by cjohnson001** [requarks/wiki#6484] [Help / Questions] [answered]
I was wondering if you guys knew how to style your wiki.js documents like wiki.js's own Docs main page:
![image](https://github.com/requarks/wiki/assets/135730808/a7197c4c-2f96-4861-82ff-b668246e9197)

I'm mostly interested in how they made the clickable Section with the emojis: 
![image](https://github.com/requarks/wiki/assets/135730808/9658fc1c-6a4b-4402-b213-66db0206a48d)

Thanks in advance!

**Comment by fnafox** [comment, marked as answer]
This is how you can achieve the desired formatting:
```
- [:globe_with_meridians: Locales *Display the wiki in a different language or enable multilingual capabilities.*](/locales)
{.links-list}
```
PS: If you click on the `Page Actions > View Source` in the top right of the docs, you will see the source and the correct formatting for the elements on the page.

**Comment by cjohnson001** [comment]
Nice, I didn't notice that view source feature! thanks so much!
